usl_pipeline/cloud_functions/main_test.py

```diff
@@ -380,10 +384,14 @@ def test_build_feature_matrix_wrf(mock_storage_client, mock_firestore_client, _)
                 {
                     # For heat, we process each WPS netcdf file as a chunk (un-tar'ed)
                     "archive_path": "gs://bucket/study_area/met_em.d03_test.nc",
-                    "feature_matrix_path": (
+                    "unscaled_feature_matrix_path": (
```

Comment:
Surprisingly my change in the study area chunk metadata code is involved into WRF too! Walt, I guess we need to discuss how to decouple Flood and WRF here...

Reply:
I ended up with using this check to separate flood from WRF: https://github.com/UrbanSystemsLab/climateiq-cnn/pull/92/files#diff-fed8bb97ec2754b34cfbc4861b38234014e6cf77913263164691eeb39c8be94eR331